content/swagger/paths.yaml

```diff
@@ -1,12 +1,3 @@
-/api/rest/v1/products:
```

Comment:
We change the order of endpoints to display first the product UUID before the product identifier